src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ScrollBarTests/UnoSamples_Tests.ScrollBar.cs

```diff
@@ -19,6 +19,7 @@ public partial class UnoSamplesTests_ScrollBar : SampleControlUITestBase
 	{
 		[Test]
 		[AutoRetry]
+		[ActivePlatforms(Platform.Browser)] // This test cannot run on Android/iOS, because the ScrollBar buttons do not react to touch (which is valid MUX behavior).
```

Comment:
Why was it working before ? Is it because the styles changed ?

Reply:
Yes, the `RepeatButton` was missing the `ShouldIgnoreTouchInput` feature